### PR TITLE
Moved guides to FAQ folder + renamed + cleaned up the overview.

### PR DESCRIPTION
**Pull request by ryanbnl**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -16,114 +15,98 @@ Member States (from the European Economic Area, which encompasses more countries
...
+The DCC specifications have been split into four repositiros:
 
-https://github.com/ehn-dcc-development/base45-php
-https://github.com/ehn-dcc-development/base45-cs
-https://github.com/ehn-dcc-development/base45-ansi-C
-https://github.com/ehn-dcc-development/base45-swift
-https://github.com/ehn-dcc-development/base45-js
-https://github.com/ehn-dcc-development/base45-java
+* [hcert-spec](https://github.com/ehn-dcc-development/hcert-spec) specifies the container and encoding formats used by the DCC.
+* [ehn-dcc-valuesets](https://github.com/ehn-dcc-development/ehn-dcc-valuesets) provides the valueset datastructures and a snapshot of the valuesets (the current versions are published on the DCCG Gateway.
```
Comment (dslmeinte) [inline readme.md:27]: Maybe mention schema repo first, and then value sets? Would be the logical as well as dependency order.

File: readme.md [hunk 2]
```diff
@@ -16,114 +15,98 @@ Member States (from the European Economic Area, which encompasses more countries
...
+You can find these repositories pinned on our github [homepage](https://github.com/ehn-dcc-development).
 
-## Verifiers
+We have a team of maintainers who monitor the above mentioned pinned repositories. If you have questions or suggestions please feel free to open an issue on the respective repository.
 
-[Explain]
+## Maintainers
 
-https://github.com/ehn-dcc-development/DGCValidator | Xamarin scanner app (Android/iOS)
+The core repoisotires are maintained a small group of people attached to EU Member States or the European Commission.
```
Comment (dslmeinte) [inline readme.md:37]: “repositories”

File: readme.md [hunk 3]
```diff
@@ -16,114 +15,98 @@ Member States (from the European Economic Area, which encompasses more countries
...
+The core repoisotires are maintained a small group of people attached to EU Member States or the European Commission. 
 
-https://github.com/ehn-dcc-development/ValidationCore | Swift/Android scanner libraries
+Repository          | Teams
+------------------- | ------------
+hcert-spec          | [hcert-spec-editors](https://github.com/orgs/ehn-dcc-development/teams/hcert-spec-editors)
+ehn-dcc-valuesets   | [ehn-valueset-editors](https://github.com/orgs/ehn-dcc-development/teams/ehn-valueset-editors)
+ehn-dcc-schema      | [ehn-schema-editors](https://github.com/orgs/ehn-dcc-development/teams/ehn-schema-editors)
+dgc-business-rules  | [ehn-business-rules-editors](https://github.com/orgs/ehn-dcc-development/teams/ehn-business-rules-editors)
+overview            | [overview-editors](https://github.com/orgs/ehn-dcc-development/teams/overview-editors)
```
Comment (dslmeinte) [inline readme.md:45]: Maybe put `overview` upfront?

Comment (dslmeinte) [reply readme.md:45]: Do you want to add repos like `dcc-business-rules-analysis`, etc. here as well?

File: readme.md [hunk 4]
```diff
@@ -16,114 +15,98 @@ Member States (from the European Economic Area, which encompasses more countries
...
-https://github.com/ehn-dcc-development/base45-java
+* [hcert-spec](https://github.com/ehn-dcc-development/hcert-spec) specifies the container and encoding formats used by the DCC.
+* [ehn-dcc-valuesets](https://github.com/ehn-dcc-development/ehn-dcc-valuesets) provides the valueset datastructures and a snapshot of the valuesets (the current versions are published on the DCCG Gateway.
+* [ehn-dcc-schema](https://github.com/ehn-dcc-development/ehn-dcc-schema) specifies the DCC schema (the specification of the data stored within the DCC).
+* [dgc-business-rules](https://github.com/ehn-dcc-development/dgc-business-rules) specifies the Business Rules framework.
 
+You can find these repositories pinned on our github [homepage](https://github.com/ehn-dcc-development).
 
-## Verifiers
+We have a team of maintainers who monitor the above mentioned pinned repositories. If you have questions or suggestions please feel free to open an issue on the respective repository.
```
Comment (dslmeinte) [inline readme.md:33]: The question is whether this team will really exist in a continuous sense after roughly June 1st. Could the EC make an email address that forwards to the relevant people, e.g. Stavros, Konstantin, etc.?

**Review by dslmeinte — CHANGES_REQUESTED** [review]

File: readme.md (hunk 4 above)
Comment (ryanbnl) [reply readme.md:33]: we do for the coming time - there will always be people maintaining it as long as it's in maintenance.

File: readme.md (hunk 3 above)
Comment (ryanbnl) [reply readme.md]: Yeah, if we finish this PR, then you can add anything I've missed? :)

**Pull request merged**